kuma/users/views.py

```diff
@@ -65,9 +65,12 @@ def ban_user(request, user_id):
                           is_active=True)
             ban.save()
             return redirect(user)
-    if user.active_ban:
-        return redirect(user)
-    form = UserBanForm()
+        else:
+            form = UserBanForm(request.POST)
```

Comment:
The form doesn't need to be reinitialized. The one that just got validated above (``form.is_valid()``) is fine. The important part is the next few lines, that don't overwrite it with an unbound form.